sqlmesh/core/context_diff.py

```diff
@@ -330,22 +331,59 @@ def requirements_diff(self) -> str:
             )
         )
 
-    def environment_statements_diff(self) -> str:
+    def environment_statements_diff(self) -> t.List[Syntax]:
+        PYTHON_ENV = "python_env"
+
+        def extract_python_env(python_env: t.Dict[str, Executable]) -> t.List[str]:
+            return [
+                v.payload if v.is_import or v.is_definition else f"{k} = {v.payload}"
+                for k, v in python_env.items()
+            ]
```

Comment:
Let's not copy-paste code like this, I'd rather extract out the logic common to `render_definition` and this method into a helper under `model/common.py` or something.